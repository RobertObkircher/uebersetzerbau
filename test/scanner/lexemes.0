end>==continue,,=)(;return var cond
->= *39->:break tail head islist
or$Bbreak   not
-->
