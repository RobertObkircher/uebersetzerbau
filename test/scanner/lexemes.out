end
>=
=
continue
,
,
=
)
(
;
return
var
cond
->
=
*
num 39
->
:
break
tail
head
islist
or
num 187
id reak
not
-
->
